lib/manager/pipenv/artifacts.ts

```diff
@@ -72,7 +72,7 @@ export async function updateArtifacts({
         image: 'renovate/python',
         tagConstraint,
         tagScheme: 'pep440',
-        preCommands: ['pip install --user pipenv'],
+        preCommands: ['pip install --user pipenv==2020.8.13'],
```

Comment:
This version should be extracted from deps instead of hard coded

Reply:
I thought of that, but did not see any place in the repo where Python dependencies are enumerated (`requirements.txt`, `Pipfile`, etc), which makes sense as this is a TypeScript codebase.  Am I just missing where that is, or are you suggesting that we should add some kind of Python dependency file like the ones I mentioned?

Reply:
This should be read from target repo, which is currently processed and fallback to latest as before.

Reply:
Thanks! Just to be explicit, we're saying the behavior should be:

- Renovate does not look for a `pipenv` version here, in its own repo
- Instead, Renovate installs the version of `pipenv` specified in the `Pipfile` of the repo Renovate is running against.  (Ie. if my repo has `pipenv==2020.8.13` tagged in `Pipfile`, Renovate should use that version when running on my repo)
- Otherwise, use the latest `pipenv` (current behavior)

Reply:
yes

Reply:
Change has been made and test coverage added